docs/admin/admission-controllers.md

```diff
@@ -197,7 +197,7 @@ We strongly recommend using this plug-in if you intend to make use of Kubernetes
 
 ### SecurityContextDeny
 
-This plug-in will deny any pod with a [SecurityContext](/docs/user-guide/security-context) that defines options that were not available on the `Container`.
+This plug-in will deny any pod that attempts to set certain escalating [SecurityContext](/docs/user-guide/security-context) fields. This should be enabled if a cluster doesn't utilize [pod security policies](/docs/user-guide/pod-security-policy) to restrict the set of values a security context can take.
```

Comment:
From what I can tell based on the code[0], certain fields aren't limited e.g. FsGroup and RunAsNonRoot.[1]

[0] https://github.com/kubernetes/kubernetes/blob/v1.6.0-rc.1/plugin/pkg/admission/securitycontext/scdeny/admission.go
[1] https://github.com/kubernetes/kubernetes/blob/v1.6.0-rc.1/pkg/api/v1/types.go#L2381